flask/testing.py

```diff
@@ -48,6 +48,11 @@ class FlaskClient(Client):
 
     preserve_context = False
 
+    def __init__(self, *args, **kwargs):
+        environ_base = kwargs.pop("environ_base", None)
+        super(FlaskClient, self).__init__(*args, **kwargs)
+        self.environ_base = environ_base
```

Comment:
I mean, currently to add your own `environ_base` you would do this:

    c = TestClient(environ_base={...})

But I only want to do this, because it's not that much of a common usecase:

    c = TestClient()
    c.environ_base = ...